examples/mouse_eye_movements.py

```diff
@@ -0,0 +1,99 @@
+"""Analyse eye movements of a mouse
+===================================
+
+Look at eye movements caused by the oculo-motor reflex.
+"""
+
+# %%
+# Imports
+# -------
+from pathlib import Path
+
+import sleap_io as sio
+from matplotlib import pyplot as plt
+
+# from movement import sample_data
+from movement.io import load_poses
+from movement.plots import plot_trajectory
+from movement.utils.vector import compute_norm
+
+# %%
+# Load the data
+# -------------
+# Define the file paths to the data.
+
+data_dir = Path.home() / "Data" / "Sepi-data" / "eye tracking"
```

Comment:
Now you could directly get it from our sample datasets (ignore me if you were already planning this).